PyInstaller/hooks/hook-firebase_admin.py

```diff
@@ -0,0 +1,11 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013-2018, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import copy_metadata
+datas = copy_metadata('google-api-core')
```

Comment:
Why does this package require meta-data from another package? Shouldn't that other package be imported then?